src/components/tutorial-meta/components/variant-list/index.tsx

```diff
@@ -0,0 +1,59 @@
+// @TODO uncomment when enabling cookies
+// import Cookies from 'js-cookie'
+import Link from 'next/link'
+import classNames from 'classnames'
+import {
+	TutorialVariant,
+	TutorialVariantOption,
+	getVariantParam,
+	getVariantPath,
+} from 'views/tutorial-view/utils/variants'
+import { useRouter } from 'next/router'
+import s from './variant-list.module.css'
+
+export default function VariantList({ variant }: { variant: TutorialVariant }) {
+	// @TODO hook this into a useVariants hook
+	const activeVariantOption = variant.options[0]
+	const VARIANT_LIST_ID = 'variant-list-label'
+	const { asPath } = useRouter()
+
+	return (
+		<div className={s.root}>
+			<label id={VARIANT_LIST_ID} className={s.label}>
+				{variant.name}
+			</label>
+			<nav>
+				<ul aria-labelledby={VARIANT_LIST_ID} className={s.list}>
+					{variant.options.map((option: TutorialVariantOption) => {
+						const variantParam = getVariantParam(variant.slug, option.slug)
+						const isActiveOption = activeVariantOption.slug === option.slug
+						return (
+							<li
+								key={option.slug}
+								aria-current={activeVariantOption.slug === option.slug}
```

Comment:
🚧 2 small things here:

- `aria-current` should be on the `<a>` elements rather than the `<li>` ones
- the value we want to set for the current element is `"page"` rather than a boolean since these are navigation links

More on `aria-current` for reference here: https://www.w3.org/TR/wai-aria-1.1/#aria-current

Reply:
Ah okay, thank you!